src/schema/v2/infiniteDiscovery/discoverArtworks.ts

```diff
@@ -21,19 +26,43 @@ export const DiscoverArtworks: GraphQLFieldConfig<void, ResolverContext> = {
   args: pageable({
     userId: { type: GraphQLString },
     limit: { type: GraphQLInt },
+    certainty: { type: GraphQLInt },
+    sort: {
+      type: new GraphQLEnumType({
+        name: "DiscoverArtworksSort",
+        values: {
+          CERTAINTY_ASC: {
+            value: "certainty",
+            description:
+              "Sort by certainty ascending. High certainty means the artwork is more likely to be recommended.",
+          },
+          CERTAINTY_DESC: {
+            value: "-certainty",
+            description:
+              "Sort by certainty descending. High certainty means the artwork is more likely to be recommended.",
+          },
+        },
+      }),
+    },
   }),
   resolve: async (_root, args, { weaviateGraphqlLoader, artworksLoader }) => {
     if (!weaviateGraphqlLoader) return
 
-    const { userId, limit = 5 } = args
+    const { userId, limit = 5, certainty = 0.5, sort } = args
```

Comment:
A somewhat arbitrary default but should be fine for testing. What this means is `weaviate` will only return `InfiniteDiscoveryArtworks` that have a `certainty` score greater than `0.5`.